### PR TITLE
Added notes about values of governable parameters is Solidity docs

### PR DESCRIPTION
**Pull request by tomaszslabon**
This PR adds some info to Solidity docs for functions that set governable parameters.
I've looked at all the functions that allow the governance to set numerical values (timeouts, delays, percentage) and they mostly seem to be well documented.

The contracts with such functions:
`AssetPool`:
* `beginWithdrawalDelayUpdate`
* `beginWithdrawalTimeoutUpdate`

`CoveragePool`:
* `beginWithdrawalDelayUpdate`
* `beginWithdrawalTimeoutUpdate`

`RiskaManagerV1`:
* `beginBondAuctionThresholdUpdate`
* `beginAuctionLengthUpdate`
* `beginLiquidationNotifierRewardUpdate`
* `beginLiquidatedNotifierRewardUpdate`

`SignerBondsUniswap`:
* `setMaxAllowedPriceImpact`
* `setSlippageTolerance`
* `setSwapDeadline`
* `setRevertIfAuctionOpen`

Closes https://github.com/keep-network/coverage-pools/issues/131

### CODE REVIEW

File: contracts/RiskManagerV1.sol [hunk 1]
```diff
@@ -332,7 +332,8 @@ contract RiskManagerV1 is IRiskManagerV1, Auctioneer, Ownable {
     }
 
     /// @notice Begins the bond auction threshold update process.
-    /// @dev Can be called only by the contract owner.
+    /// @dev Can be called only by the contract owner. The value of the
+    ///      threshold should not be greater than 100.
     /// @param _newBondAuctionThreshold New bond auction threshold in percent
     function beginBondAuctionThresholdUpdate(uint256 _newBondAuctionThreshold)
```
Comment (tomaszslabon) [inline contracts/RiskManagerV1.sol:338]: Here we are setting a percentage value.
We could consider checking `_newBondAuctionThreshold` (it should not be greater than 100).

Comment (pdyraga) [reply contracts/RiskManagerV1.sol:338]: Let's do this.

Comment (tomaszslabon) [reply contracts/RiskManagerV1.sol:338]: Done in https://github.com/keep-network/coverage-pools/pull/160/commits/db6530d23902d656cb1c244a71203c4be7077f3d.

**Comment by pdyraga** [comment]
@tomaszslabon I wanted to add more information to docs regarding what the governance should take into account when updating parameters. I figured it will be faster if I just push. Let me know what you think about the proposed changes, if you are good with them, we can merge.

**Comment by tomaszslabon** [comment]
> @tomaszslabon I wanted to add more information to docs regarding what the governance should take into account when updating parameters. I figured it will be faster if I just push. Let me know what you think about the proposed changes, if you are good with them, we can merge.

@pdyraga I looked at it and it seems ok. I think we can merge it.

**Review by pdyraga — APPROVED** [review]

**Pull request merged**